spec/acceptance/basic_spec.rb

```diff
@@ -1,7 +1,19 @@
 require 'spec_helper_acceptance'
 
 describe 'basic installation' do
-  let(:pp) { 'include pulpcore' }
+  let(:pp) {
+    'include pulpcore'
+    "class { 'postgresql::globals':
```

Comment:
This needs to come before including pulpcore since this way of setting parameters is compile order dependent (unlike hiera).